### DISCUSSION TITLE
Hiarchical Tags

### DISCUSSION THREAD

**Discussion by AnthonySummers** [rswag/rswag#602] [General]
Is there a way to have tags so we can organize by at least two levels instead of one for example:

- restaurant
      - food
          ->  /api/create_food
      - drink 
          ->  /api/create_drink
 - management
       - employees
    
    etc...

**Comment by romanblanco** [comment]
@AnthonySummers Nested tags are not supported by the OpenAPI Specification. (see https://github.com/OAI/OpenAPI-Specification/issues/1367)
